mars/dataframe/tests/test_core.py

```diff
@@ -201,6 +201,101 @@ def test_to_frame_or_series(setup):
     pd.testing.assert_series_equal(raw.to_series(name='new_name'), result)
 
 
+def test_assign():
```

Comment:
Please add a parameter `setup` as a parameter, `setup` is a pytest fixture and it can make sure each test use a single test cluster. Refer to https://github.com/mars-project/mars/blob/0c9114d1fffe56af7e3a3da01c7534e9597237a2/mars/dataframe/tests/test_core.py#L138

Reply:
Done